driver-sync/src/test/functional/com/mongodb/client/unified/UnifiedTest.java

```diff
@@ -242,7 +245,14 @@ public void shouldPassAllOutcomes() {
         }
 
         if (definition.containsKey("expectLogMessages")) {
-            compareLogMessages(rootContext, definition);
+            ArrayList<LogMatcher.Tweak> tweaks = new ArrayList<>(singletonList(
```

Comment:
Alternative proposal: skip the new tests when connecting via unix server address.  Something like:

```
assumeTrue(ServerAddressHelper.createServerAddress(ClusterFixture.getConnectionString().getHosts().get(0)) 
      instanceof UnixServerAddress);
```

in the `ServerSelectionLoggingTest` constructors.

Reply:
(If I'm reading it write that the tweaks are for the unix server address task and not the load balancer task)

Reply:
> skip the new tests when connecting via unix server address

I thought about it, and decided that it's better to have them being run, then not. After all, I had to add conditional formatting (`[`, `]`) to the log messages format specifically for the Unix domain sockets (the spec does not cover that).

> the tweaks are for the unix server address task and not the load balancer task

Right, `serverless-test` has nothing to do with `LogMatcher.Tweak`. Currently, two tweaks are applied:

- unconditionally skip matching the `operation` entry;
- skip matching the `serverPort` entry if Unix domain socket is used.